.github/actions/Tests/Unattended/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+bash unattended_scripts/open-distro/unattended-installation/unattended-installation.sh
+cat /var/log/wazuh-unattended-installation.log
+sleep 60
+cd ~
+/usr/testing/bin/pytest --tb=long /test_unattended.py -v
```

Comment:
Why is this added?

Reply:
This was a bad commit, I already deleted this folder.